tsconfig.app.json

```diff
@@ -3,6 +3,7 @@
   "include": ["env.d.ts", "src/**/*", "src/**/*.vue"],
   "exclude": ["src/**/__tests__/*"],
   "compilerOptions": {
+    "lib": ["ES2021"],
```

Comment:
Making sure that we can use latest features of ES. In my case I needed `String.prototype.replaceAll()`